entity-framework/core/what-is-new/ef-core-9.0/whatsnew.md

```diff
@@ -562,6 +562,124 @@ protected override void Up(MigrationBuilder migrationBuilder)
 
 ## Model building
 
+<a name="auto-compiled-models"></a>
+
+### Auto-compiled models
+
+> [!TIP]
+> The code shown here comes from the [NewInEFCore9.CompiledModels](https://github.com/dotnet/EntityFramework.Docs/tree/main/samples/core/Miscellaneous/NewInEFCore9.CompiledModels/) sample.
+
+Compiled models can improve startup time for applications with large models--that is entity type counts in the 100s or 1000s. In previous versions of EF Core, a compiled model had to be generated manually, using the command line. For example:
+
+```dotnetcli
+dotnet ef dbcontext optimize
+```
+
+After running the command, a line like, `.UseModel(MyCompiledModels.BlogsContextModel.Instance)` must be added to `OnConfiguring` to tell EF Core to use the compiled model.
+
+Starting with EF9, this `.UseModel` line is no longer needed. Instead, the compiled model will be detected and used automatically. This can be seen by having EF log whenever it is building the model. Running a simple application then shows EF building the model when the application starts:
+
+```output
+Starting application...
+>> EF is building the model...
+Model loaded model with 2 entity types.
+```
+
+The output from running `dotnet ef dbcontext optimize` on the model project is:
+
+```output
+PS D:\code\EntityFramework.Docs\samples\core\Miscellaneous\NewInEFCore9.CompiledModels\Model> dotnet ef dbcontext optimize
+
+Build succeeded in 0.3s
+
+Build succeeded in 0.3s
+Build started...
+Build succeeded.
+>> EF is building the model...
+>> EF is building the model...
+Successfully generated a compiled model, it will be discovered automatically, but you can also call 'options.UseModel(BlogsContextModel.Instance)'. Run this command again when the model is modified.
+PS D:\code\EntityFramework.Docs\samples\core\Miscellaneous\NewInEFCore9.CompiledModels\Model> 
+```
+
+Notice that the log output indicates that the _model was built when running the command_. If we now run the application again, after rebuilding but without making any code changes, then the output is:
+
+```output
+Starting application...
+Model loaded model with 2 entity types.
```

Comment:
We should fix this string

Reply:
@AndriySvyryd Fix in what way?

Reply:
Oh. Can't see the forest for the trees... :-)